### PR TITLE
Update with some of the options that are avalbile

### PR DESCRIPTION
**Pull request by plessbd**
These should have been added when the options were added, but they were not, and I need to keep reminding myself what they are, so I added some help.

### CODE REVIEW

File: bin/xdmod-ingestor [hunk 1]
```diff
@@ -333,6 +333,12 @@ Usage: xdmod-ingestor [-v]
     --debug
         Output debug level and above log messages.
 
+    --aggregate *realm*
+        If specified optional realm to aggregate (job, cloud, storage)
+
+    --datatype [openstack, genericcloud, storage]
```
Comment (smgallo) [inline bin/xdmod-ingestor]: The OCD in me wants this to be `cloud-openstack, cloud-generic, storage`

**Comment by jpwhite4** [comment]
What about the rest of the options????!!!!?!?!?!?!?!??!?!?!?!?!?

File: bin/xdmod-ingestor [hunk 2]
```diff
@@ -333,6 +333,12 @@ Usage: xdmod-ingestor [-v]
     --debug
         Output debug level and above log messages.
 
+    --aggregate *realm*
+        If specified optional realm to aggregate (job, cloud, storage)
```
Comment (jtpalmer) [inline bin/xdmod-ingestor]: You should indicate that the realm is optional and that if you do specify the realm that you need to use `=` as the separator and not space.

Comment (smgallo) [reply bin/xdmod-ingestor]: @jtpalmer Is that the case with the other options as well?

File: bin/xdmod-ingestor [hunk 3]
```diff
@@ -333,6 +333,12 @@ Usage: xdmod-ingestor [-v]
     --debug
         Output debug level and above log messages.
 
+    --aggregate *realm*
```
Comment (jtpalmer) [inline bin/xdmod-ingestor]:
```suggestion
    --aggregate[=*realm*]
```

File: bin/xdmod-ingestor [hunk 4]
```diff
@@ -333,6 +333,12 @@ Usage: xdmod-ingestor [-v]
     --debug
         Output debug level and above log messages.
 
+    --aggregate[=*realm*]
+        If specified optional realm to aggregate (job, cloud, storage)
+
+    --datatype [openstack, genericcloud, storage]
+        Ingest one specific datatype
```
Comment (jtpalmer) [inline bin/xdmod-ingestor]:
```suggestion
        Ingest one specific datatype.
```

File: bin/xdmod-ingestor (hunk 2 above)
Comment (jtpalmer) [reply bin/xdmod-ingestor]: This is the only option that optionally accepts an argument.

Comment (smgallo) [reply bin/xdmod-ingestor]: I'm wondering why this one requires an `=` rather than a space like the other arguments.

Comment (jtpalmer) [reply bin/xdmod-ingestor]: And the requirement to use `=` for optional arguments is from my reading of the [`getopt`](http://php.net/getopt) documentation:

```
Note: Optional values do not accept " " (space) as a separator.
```

File: bin/xdmod-ingestor (hunk 1 above)
Comment (plessbd) [reply bin/xdmod-ingestor]: That would be @eiffel777 who chose those, I'm just documenting it ;)

**Comment by plessbd** [comment]
@jpwhite4 if you provide me a list of the other options in the format of

```
--optionName *optionalOptionvalue*
    Description of option
```

I am sure I can add them in ;)

**Pull request merged**